packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -4149,7 +4155,14 @@ class IndexedStack extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     final List<Widget> wrappedChildren = List<Widget>.generate(children.length, (int i) {
-      return Visibility.maintain(visible: i == index, child: children[i]);
+      return Visibility(
+        visible: i == index,
+        maintainInteractivity: maintainFocusability,
```

Comment:
Maybe we can match with the same name for consistency? :)

```suggestion
        maintainInteractivity: maintainInteractivity,
```